### DISCUSSION TITLE
Dağıtılan Ödülü Artırmak...

### DISCUSSION THREAD

**Discussion by Skips1** [IBAX-io/go-ibax#971] [Q&A]
Ne düşünüyorsunuz bilmiyorum fakat bence İbax ın arzına göre çok ama çok az airdrop dağıtıyorlar zaten yarısını referans kasanlara vericekler. Bence Dağıtılan Coin artırılmalı benim gibi düşünen yokmu gerçekten ???

![wallpaper-3584230_1920](https://user-images.githubusercontent.com/98268472/151225982-3600da9b-339a-4597-a7e7-7a9c2c698a8f.jpg)

**Comment by ghost** [comment]
+1
